src/Adapter/Product/Combination/CommandHandler/UpdateCombinationStockHandler.php

```diff
@@ -43,22 +45,38 @@ final class UpdateCombinationStockHandler implements UpdateCombinationStockHandl
      */
     private $combinationStockUpdater;
 
+    /**
+     * @var MovementReasonRepository
+     */
+    private $movementReasonRepository;
+
     /**
      * @param CombinationStockUpdater $combinationStockUpdater
+     * @param MovementReasonRepository $movementReasonRepository
      */
     public function __construct(
-        CombinationStockUpdater $combinationStockUpdater
+        CombinationStockUpdater $combinationStockUpdater,
+        MovementReasonRepository $movementReasonRepository
     ) {
         $this->combinationStockUpdater = $combinationStockUpdater;
+        $this->movementReasonRepository = $movementReasonRepository;
     }
 
     /**
      * {@inheritdoc}
      */
     public function handle(UpdateCombinationStockCommand $command): void
     {
+        $deltaQuantity = null;
```

Comment:
You need to check if `$command->getDeltaQuantity() !== 0` here, because the command allows this value, and it's the default one in the input field So there are risks the form will fail because of the VO internal exception

Reply:
The same modification will probably be needed in the product handler (if you made the same modification, although it can be done in another PR if you prefer, but I belive you already started modifying the udpater)